project/categories/routes.py

```diff
@@ -0,0 +1,156 @@
+# Modules from Flask Library
+from flask import Flask, render_template, request, flash, Blueprint
+from flask import session as login_session
+from flask import make_response, redirect, jsonify, url_for
+# Access database
+from project.models import Category, Product, User
+from project.db import DBConnector
+# Services
+from project.services.categories import CategoryService
+from project.services.auth import AuthService
+from project.services.products import ProductService
+
+import random
+import string
+import httplib2
+import json
+import requests
+
+
+# Create Blueprint 'categories'
+categories = Blueprint('categories', __name__)
+
+# Connect to the database, create session
+session = DBConnector().get_session()
+
+# Instantiate services
+category_service = CategoryService()
+product_service = ProductService()
+auth_service = AuthService()
+
+
+@categories.route('/')
+@categories.route('/categories/')
+def showCategories():
+    """App route function for main page to show all categories."""
+    return render_template('index.html',
+                           categories=category_service.get_all_categories(),
+                           isLogin=auth_service.is_user_authorized(),
+                           latestProducts=(product_service
+                                           .get_latest_products(8)))
```

Comment:
Same here: assign `8` to a constant, for ex `DEFAULT_CATEGORY_PRODUCT_NUMBER` and use the constant.